packages/office-ui-fabric-react/src/components/Icon/Icon.base.tsx

```diff
@@ -26,7 +26,12 @@ export class IconBase extends React.Component<IIconProps, IIconState> {
     const { className, styles, iconName, imageErrorAs, theme } = this.props;
     const isPlaceholder = typeof iconName === 'string' && iconName.length === 0;
     const isImage = this.props.iconType === IconType.image || this.props.iconType === IconType.Image || !!this.props.imageProps;
-    const { iconClassName, children } = getIconContent(iconName);
+    const iconContent = getIconContent(iconName);
+
+    const { iconClassName, children } = iconContent || {
```

Comment:
Similar to above, just do `const { iconClassName, children } = iconContent || {};`

Reply:
Made the change